presto-orc/src/main/java/com/facebook/presto/orc/reader/MapFlatBatchStreamReader.java

```diff
@@ -158,13 +158,7 @@ public Block readBlock()
         else {
             nullVector = new boolean[nextBatchSize];
             int nullValues = presentStream.getUnsetBits(nextBatchSize, nullVector);
-            if (nullValues == nextBatchSize) {
-                for (int i = 0; i < inMapStreams.size(); i++) {
-                    inMapVectors[i] = null;
-                    totalMapEntries += nextBatchSize;
```

Comment:
Just for my understanding, I believe this is the real bug. But inMapVectors[i] is already null as it was just allocated in the top so this code is no-op,so this entire code is removed.

Reply:
exactly